index.md

```diff
@@ -331,16 +331,16 @@ _Impartial_: All parties have opportunity to communicate their perspective on th
 
 _Sensitive_: All grievances will be dealt with sensitively and professionally by staff experienced in these matters.
 
-_Free of retaliation or victimization_: Staff involved in a grievance shall not victimized by anyone for coming forward with the grievance or for assisting in its resolution. Any victimization or retaliation may result in disciplinary action including termination of the perpetrator. 
+_Free of retaliation or victimization_: Staff involved in a grievance shall not victimized by anyone for coming forward with the grievance or for assisting in its resolution. Any victimization or retaliation may result in disciplinary action from the hiring and compensation committee, with input from your manager and the HR specialist, including termination of the perpetrator. 
```

Comment:
Looks like this should be, "shall not victimize," rather than, " shall not victimized by."